### PR TITLE
add facebook and reddit logging

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

File: src/utils/events.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Call once on a new user's first tab.
```
Comment (kmjennison) [inline src/utils/events.js:1]: Move comment to above func.

File: src/utils/__tests__/reddit-analytics.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+describe('reddit-analytics', () => {
+  test('calls Reddit analytics library as expected', () => {
+    const redditAnalytics = require('src/utils/reddit-analytics').default
+    isClientSide.mockReturnValue(true)
+    redditAnalytics('track', 'SignUp')
+    expect(window.rdt).toHaveBeenCalledWith('track', 'SignUp')
+  })
+
+  test('does not call or throw if serverside', () => {
+    const fbAnalytics = require('src/utils/facebook-analytics').default
```
Comment (kmjennison) [inline src/utils/__tests__/reddit-analytics.test.js:23]: Use correct module here.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Nice work! Only tiny changes. I also manually tested with a user set to zero tabs and saw the events fire.

**Review by kmjennison — APPROVED** [review]
LGTM 🍝
